scripts/nx-release.ts

```diff
@@ -116,36 +115,6 @@ const LARGE_BUFFER = 1024 * 1000000;
   if (options.dryRun) {
     console.warn('Not Publishing because --dryRun was passed');
   } else {
-    // If publishing locally, force all projects to not be private first
```

Comment:
This is removed because `@nx/js:` handles this?

Reply:
No it’s because we agreed that the only workaround for this now was to manually set the targets instead, so I went to do that and realised we don’t have any private projects. If you want me to restore it in case we ever do something like that again in the future I can

Reply:
I have restored the behaviour in case we ever need it again in future, and updated the log message to more explicitly cover the fact that a target will now need to be added manually for the private packages